package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "css-loader": "^7.1.2",
     "css-minimizer-webpack-plugin": "^7.0.0",
+    "dropzone": "^6.0.0-beta.2",
```

Comment:
Only one change - can you change the version of this package to [5.9.3](https://www.npmjs.com/package/dropzone/v/5.9.3)? I'm wary about using a beta version.

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/254/commits/8e8335135f28f348ed7f94d0a632d960746d12d9